src/test/java/com/puppycrawl/tools/checkstyle/checks/imports/UnitImportControlTest.java

```diff
@@ -0,0 +1,61 @@
+////////////////////////////////////////////////////////////////////////////////
+// checkstyle: Checks Java source code for adherence to a set of rules.
+// Copyright (C) 2001-2018 the original author or authors.
+//
+// This library is free software; you can redistribute it and/or
+// modify it under the terms of the GNU Lesser General Public
+// License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+//
+// This library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public
+// License along with this library; if not, write to the Free Software
+// Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+////////////////////////////////////////////////////////////////////////////////
+
+package com.puppycrawl.tools.checkstyle.checks.imports;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Before;
+import org.junit.Test;
+
+public class UnitImportControlTest {
+
+    private final PkgImportControl icRoot = new PkgImportControl("com.kazgroup.courtlink",
+            false, MismatchStrategy.DISALLOWED);
+
+    private final UnitImportControl icClass = new UnitImportControl(icRoot, "MyClass",
+            false);
```

Comment:
variable name should be "Unit".

do we really need "ic"(import control) prefix ?looks weird , took me some time to decode it.
What about unitControl or rootControl.
will looks good "rootControl.addImportRule(...)"

Reply:
> do we really need "ic"(import control) prefix ?

I was just following pattern of original test as package and units are similar. See https://github.com/checkstyle/checkstyle/blob/90f20e09869c71eb22190ad9c1d46d5deec324a5/src/test/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportControlTest.java#L30

Reply:
lets stop expanding of this pattern now, 
do not change other places , but please name properly in your new code.

Reply:
Done.